src/hydra_zen/funcs.py

```diff
@@ -45,10 +45,24 @@ def zen_processing(
     _zen_target: str,
     _zen_partial: bool = False,
     _zen_exclude: _typing.Sequence[str] = tuple(),
+    _zen_wrappers: _typing.Union[str, _typing.Sequence[str]] = tuple(),
     **kwargs,
 ):
+    if isinstance(_zen_wrappers, str) or not isinstance(
+        _zen_wrappers, _typing.Sequence
+    ):
+        _zen_wrappers = (_zen_wrappers,)
+
+    # flip order: first wrapper listed should be called first
+    wrappers = tuple(
+        get_obj(path=z) if isinstance(z, str) else z for z in _zen_wrappers
+    )[::-1]
+
     obj = get_obj(path=_zen_target)
 
+    for wrapper in wrappers:
+        obj = wrapper(obj)
+
```

Comment:
Your comment "first wrapper listed should be called first" does not seem to agree with the result:
```python
>>> obj = 123
>>> def first(x):
...     print("first")
...     return x
...
>>> def second(x):
...     print("second")
...     return x
...
>>> _zen_wrappers = [first, second]
>>> wrappers = tuple(
...     get_obj(path=z) if isinstance(z, str) else z for z in _zen_wrappers
... )[::-1]
>>> for wrapper in wrappers:
...     obj = wrapper(obj)
...
second
first
>>> # the second wrapper listed is called first
```

Reply:
Ah, yep! Good catch.